src/main/webapp/submit_a_question.html

```diff
@@ -0,0 +1,29 @@
+<!--
```

Comment:
Me and Kai created a file for this. Also I am working on the functionalities so I prefer not to substitute because we already named the ID's of the components and I am using them in the script.js.

Reply:
What would you like me to do, then? Can you give me the file names/parts of the aspects you want me to keep? You can send screenshots through Chat if this helps